go.mod

```diff
@@ -4,8 +4,9 @@ go 1.19
 
 require (
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/terraform-plugin-go v0.18.0
+	github.com/hashicorp/terraform-plugin-go v0.18.1-0.20230830121453-fd948ecd32c7
```

Comment:
This will be updated to terraform-plugin-go v0.19.0 pending https://github.com/hashicorp/terraform-plugin-go/pull/311 review, merge, and release.